sdks/python/apache_beam/tools/distribution_counter_microbenchmark.py

```diff
@@ -23,11 +23,16 @@
   python -m apache_beam.tools.distribution_counter_microbenchmark
 """
 
+from __future__ import absolute_import
+from __future__ import division
 from __future__ import print_function
 
 import random
 import sys
 import time
+from builtins import range
+
+from past.utils import old_div
```

Comment:
Can you replace old_div with the new Python 3 division? Thanks.

Reply:
Fixed in new commit